Assets/HOLOMEProject/Script/FbxLoader.cs

```diff
@@ -12,6 +12,8 @@ public class FbxLoader : MonoBehaviourPunCallbacks
     private string gameObjectName;
     public GameObject Food;
     public GameObject Brush;
+    public ParticleSystem Shower;
```

Comment:
```suggestion
    public GameObject Shower;
```
ここの型って`GameObject`ではない？